app/src/main/java/com/pizzapp/model/pizza/Size.java

```diff
@@ -5,15 +5,17 @@
 public class Size implements Serializable {
     private String name;
     private double price;
+    private double dimension;
```

Comment:
What is the dimension objects purpose?

Reply:
The dimension is to specify the actual size of the pizza (in inches, as shown in the mockup). It was previously was hard-coded, so I added it to the object.